### ISSUE TITLE
Edge/JAWS 2019 Aria-describedby is not read.

### ISSUE THREAD

**Issue by livijohnson** [FreedomScientific/VFO-standards-support#512]
When in Edge, tabbing through a form containing aria-describedby text like below, JAWS 2019 does not read the aria-describedby text:

```
<label for="test_input">Testing</label>
<input type="text" id="test_input" aria-describedby="desc"/>
<p id="desc">Aria Described By</p>
```

Expected Result: JAWS should read something like "Testing edit use JAWS key plus alt plus R to read the descriptive text Aria described By type and text.
Actual Result: JAWS reads "Testing edit type and text.” 

I’m using Edge version 88.0.705.63 and JAWS Professional Edition version 2019.1907.42 ILM. OS is Windows 10 Version 10.0.18363 Build 18363

Thanks,
Olivia

**Comment by JAWS-test** [comment]
The problem does not occur in JAWS 2020 and JAWS 2021. The issue occurs with Edge Chromium with JAWS 2019 and other older JAWS versions, although Edge correctly submits the description in the Accessibility API. The issue cannot be fixed even with modified JAWS settings (INS+6).

**Comment by stevefaulkner** [comment]
thanks @JAWS-test, closing this as older versions of JAWS will not be fixed.

**Comment by livijohnson** [comment]
Good to know. Thanks for your response!
